x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -169,6 +170,45 @@ export const getPackageUsageStats = async ({
   };
 };
 
+export async function getPackagePermissions(
```

Comment:
It would be nice to have a test suite for this which allows to specify a package and have a gold file written of the expected permission output. This will make it much easier to verify if the code does what it should do. It also allows to add more packages for testing if needed to cover special cases and create golden files for it.